docs/en/sql-reference/statements/create/table.md

```diff
@@ -53,12 +53,32 @@ Creates a table with the same result as that of the [table function](../../../sq
 CREATE TABLE [IF NOT EXISTS] [db.]table_name ENGINE = engine AS SELECT ...
 ```
 
-Creates a table with a structure like the result of the `SELECT` query, with the `engine` engine, and fills it with data from SELECT.
+Creates a table with a structure like the result of the `SELECT` query, with the `engine` engine, and fills it with data from `SELECT`.
 
-In all cases, if `IF NOT EXISTS` is specified, the query won’t return an error if the table already exists. In this case, the query won’t do anything.
+If the table already exists and `IF NOT EXISTS` is specified, the query won’t do anything.
 
 There can be other clauses after the `ENGINE` clause in the query. See detailed documentation on how to create tables in the descriptions of [table engines](../../../engines/table-engines/index.md#table_engines).
 
+!!! warning "Warning"
+    The `SELECT` results must match the table description, otherwise an exception is thrown.
+
+**Example**
+
+Query:
+
+``` sql
+CREATE TABLE t1 (x String) ENGINE = Memory AS SELECT 1;
```

Comment:
But `SELECT` result does not match the table description is this example: `SELECT` result is `("1" UInt8)` and table description is `(x String)`.

Reply:
Fixed it. I got it wrong at first. @tavplubix